manuscript/03-01-First-steps-with-List-syntax.md

```diff
@@ -4,37 +4,33 @@
 
 У ClojureScript список, першим елементом якого є функція, використовується для виклику функції. У наступному прикладі ми викликаємо функцію додавання із трьома аргументами. Зверніть увагу на те, що у мовах родини Lisp знаком `+` позначають не оператор, а функцію. Операторів у Lisp немає - лише функції.
 
-[source, clojure]
-----
+```clojure
```

Comment:
@denysdovhan I believe both `clojure` and `cljs` highlighting are the same. I'm usually using `clojure`.